client/my-sites/checkout/checkout/index.jsx

```diff
@@ -387,13 +393,17 @@ export class Checkout extends React.Component {
 		}
 
 		if ( cart.create_new_blog ) {
-			return `/checkout/thank-you/no-site/${ receiptId }`;
+			return `${ signupDestination }/${ receiptId }`;
 		}
 
 		if ( ! selectedSiteSlug ) {
 			return '/checkout/thank-you/features';
 		}
 
+		if ( this.props.isJetpackNotAtomic ) {
```

Comment:
Jetpack plan purchases are not shown the Gsuite and Concierge upsell nudges, so it makes sense to move this here at the top of the function.